dataframe-csv/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/DelimCsvTsvTests.kt

```diff
@@ -0,0 +1,657 @@
+package org.jetbrains.kotlinx.dataframe.io
+
+import io.kotest.assertions.throwables.shouldNotThrowAny
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.collections.shouldContainInOrder
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.shouldBe
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.LocalDateTime
+import org.intellij.lang.annotations.Language
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.api.allNulls
+import org.jetbrains.kotlinx.dataframe.api.convert
+import org.jetbrains.kotlinx.dataframe.api.dataFrameOf
+import org.jetbrains.kotlinx.dataframe.api.group
+import org.jetbrains.kotlinx.dataframe.api.groupBy
+import org.jetbrains.kotlinx.dataframe.api.into
+import org.jetbrains.kotlinx.dataframe.api.isEmpty
+import org.jetbrains.kotlinx.dataframe.api.print
+import org.jetbrains.kotlinx.dataframe.api.schema
+import org.jetbrains.kotlinx.dataframe.api.toStr
+import org.junit.Test
+import java.io.File
+import java.io.StringWriter
+import java.math.BigDecimal
+import java.net.URL
+import java.util.Locale
+import java.util.zip.GZIPInputStream
+import kotlin.reflect.KClass
+import kotlin.reflect.typeOf
+
+@Suppress("ktlint:standard:argument-list-wrapping")
+class DelimCsvTsvTests {
```

Comment:
We already have a lot of CSV tests in core tests. Do you copy them here or this is a completely written from scratch? I understand that API is different, but at least for default case when only path argument is supplied, it'd be good to compare results and to be able to say they're identical or not

Reply:
yes I copied all of them and added a couple more, so I could see they behave the same.

Reply:
and actually, I changed the default arguments such that the behavior of not specifying any additional settings is as close as possible as the old behavior. Things like `allowMissingColumns` ans `ignoreExcessColumns` for instance.